### ISSUE TITLE
v3.0 - unable to disable modules

### ISSUE THREAD

**Issue by ingro** [omarkhatibco/tailwind-css-variables#15]
Hello, I've updated the plugin to the 3.0 version but I think I found out a bug when you try to disable modules.
Basically my config is something like this:

```javascript
require('tailwind-css-variables')({
    colors: 'color',
    screens: false,
    fontFamily: false,
    fontSize: false,
    fontWeight: false
})
```

But the modules with `false` are not disabled, they are just prefixed with "false":

![tailwind-css-variables](https://user-images.githubusercontent.com/879431/113393527-17964180-9397-11eb-94fe-1c24924561b0.png)

This worked fine with the previous version.

**Comment by omarkhatibgg** [comment]
Hi @ingro,

I just fixed it in the v3.0.1.

Thanks for reporting!

**Comment by ingro** [comment]
Thanks for the fast fix! :)
